Movement/Signoffs/Revamp Chassis and Motor Signoff.md

```diff
@@ -0,0 +1,99 @@
+# Function of the subsystem:
```

Comment:
Be brief. There's no reason to write lengthy paragraphs without having a lot to say.